### DISCUSSION TITLE
FEL -> MEL or FEL -> P8?

### DISCUSSION THREAD

**Discussion by nateify** [quietvoid/dovi_tool#211] [Q&A] [answered]
I have a Hisense Android TV which is DV compatible but appears to be unable to play DV P7 FEL properly in MKV container (not tested any other container). I tested it with the app Just Player as well as a special Kodi build with DV support. The TV does not recognize DV, and colors are inaccurate.

However, I have P7 MEL rips (still in MKV), and the TV detects DV in both apps and colors are correct.

It seems the tool is capable of both FEL to MEL and FEL to P8 via modes 1 and 2, is there any reason to pick one over the other in terms of image quality?

**Comment by quietvoid** [comment, marked as answer]
There should be no difference in the final image.

Some devices may have more success with profile 8.
